_posts/misceleneous/2018-07-16-add-infrastructure-for-vblank-and-page-flip-events-simulated-via-hrtimer-in-vkms.md

```diff
@@ -24,7 +24,6 @@ as Figure 1 describes.
 
 {% include image-post.html
   src="posts/tdd-drm.png"
-  no-anchor="true"
```

Comment:
Why did you drop no-anchor here? Do I need to drop it in all of the other posts?

Reply:
I dropped no-anchor because it conflicts with the animation of the popup images, as I haven't seen the frequent use of no-anchor I think it's a good idea to remove it so there is no conflict in the pop-up images.